views/order-process.html.twig

```diff
@@ -135,6 +135,20 @@ $(document).ready(function () {
     var orderView = new PMI.views['OrderSubPage']({
         el: $("body")
     });
+
+    $('.process-form button[type="submit"]').on('click', function () {
```

Comment:
May not matter in a typical workflow, but what if a user presses enter while in a field to submit the form? (We may have that blocked elsewhere.)

Reply:
We have a `disable-enter` class in the form, which prevents triggering enter.